app/views/Moderation/Schools/index.tsx

```diff
@@ -201,7 +201,7 @@ function Schools(props: Props) {
                 messageShown
                 emptyMessage={(
                     <EmptyMessage
-                        message="Couldn't find any user"
+                        message="Couldn\'t find any user"
```

Comment:
there is a typo in Couldn't find any user

Reply:
Any reason these are not translated?